### ISSUE TITLE
BeatmapParser fails to parse hit objects beatmap 197279

### ISSUE THREAD

**Issue by ThePooN** [mrflashstudio/OsuParsers#19]
BeatmapParser fails to parse hit objects of this beatmap: https://osu.ppy.sh/beatmapsets/68075#osu/197279

Stack trace:
```
System.IndexOutOfRangeException: Index was outside the bounds of the array.
   at OsuParsers.Decoders.BeatmapDecoder.ParseHitObjects(String line)
   at OsuParsers.Decoders.BeatmapDecoder.ParseLine(String line)
   at OsuParsers.Decoders.BeatmapDecoder.Decode(IEnumerable`1 lines)
   at OsuParsers.Decoders.BeatmapDecoder.Decode(String path)
```

Tested on latest master.

**Comment by mrflashstudio** [comment]
Should be fixed in 25a37e9

**Comment by ThePooN** [comment]
thanks for the quick fix!
